Ports/.hosted_defs.sh

```diff
@@ -27,6 +27,7 @@ if [ "$SERENITY_TOOLCHAIN" = "Clang" ]; then
     export SERENITY_TOOLCHAIN_BINDIR="${SERENITY_SOURCE_DIR}/Toolchain/Local/clang/bin"
     export CC="${SERENITY_ARCH}-pc-serenity-clang"
     export CXX="${SERENITY_ARCH}-pc-serenity-clang++"
+    export LD="${SERENITY_TOOLCHAIN_BINDIR}/ld.lld"
```

Comment:
Just keeping this exported even when we do a `host_env` will probably break on macOS. 

We should probably `export HOST_LD="${LD:=}"` to default it to 'not set'. Normally you wouldn't set that variable for just normal host programming.

Reply:
Thank you, as discussed on Discord, I've set it to `ld` if it was not defined.